p/themes/Origine/origine.css

```diff
@@ -330,6 +330,8 @@ a.btn {
 	box-shadow: 3px 3px 3px #ddd;
 	font-size: 0.8rem;
 	text-align: left;
+	max-height: 60vh;
+	overflow: hidden auto;
```

Comment:
That's not valid, is it?

Reply:
vh is relative to 1% screen height: https://www.w3schools.com/cssref/css_units.asp
true … i'm not sure if overflow-x and overflow-y need to be defined separately for other browsers …

Reply:
I see, this overflow shorthand is very recent so I wasn't familiar with it. It's only been implemented properly in Fx 63, which came out last week.

https://www.fxsitecompat.com/en-CA/docs/2018/overflow-shorthand-syntax-has-been-updated-to-swap-2-values/

The official line is IE11, but personally I care more about Opera 12. ;-)

Reply:
uh they swapped it from firefox 61/62 to 63 … true … better 2 explicit lines?